libraries/engage/account/actions/fetchCustomer.js

```diff
@@ -15,6 +15,7 @@ export const fetchCustomerData = () => async (dispatch) => {
   try {
     const { customer } = await (new PipelineRequest('shopgate.customer.getCustomer')
       .setTrusted()
+      .setErrorBlacklist(['EINVALIDCALL'])
```

Comment:
There is a constant for that which you can import via
```js
import { EINVALIDCALL } from '@shopgate/engage/core/constants';
```
Since this feedback is so small, i've decided to apply this little change by myself.